demos/defender/defender_demo_json/demo_config.h

```diff
@@ -42,7 +42,7 @@
 #endif
 
 #ifndef LIBRARY_LOG_LEVEL
-    #define LIBRARY_LOG_LEVEL    LOG_INFO
+    #define LIBRARY_LOG_LEVEL    LOG_DEBUG
```

Comment:
Probably better to keep as info to keep the output readable?

Reply:
Yes, this change was just an artifact of testing.

Reply:
So should be changed back?

Reply:
Yes, changed it back.